cmd/limactl/copy.go

```diff
@@ -43,23 +44,33 @@ func copyAction(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	quiet, err := cmd.Flags().GetBool("quiet")
+	if err != nil {
+		return err
+	}
+
 	arg0, err := exec.LookPath("scp")
 	if err != nil {
 		return err
 	}
+
 	instances := make(map[string]*store.Instance)
 	scpFlags := []string{}
 	scpArgs := []string{}
+
 	debug, err := cmd.Flags().GetBool("debug")
 	if err != nil {
 		return err
 	}
-	if debug {
+
+	if debug && !quiet {
```

Comment:
I don't think the `--quiet` option should override `--debug`. It should only apply to the output of the ssh command itself.